src/command/jc.jl

```diff
@@ -0,0 +1,140 @@
+# JuliaCon.debugmode(true)
+
+function commander(c::Client, m::Message, ::Val{:julia_con})
+    startswith(m.content, COMMAND_PREFIX * "con") || return
+
+    regex = Regex(COMMAND_PREFIX * raw"con( help| juliacon2021| now| today)? *(.*)$")
+
+    matches = match(regex, m.content)
+
+    if matches === nothing || matches.captures[1] ∈ (" help", nothing)
+        help_commander(c, m, :julia_con)
+    elseif matches.captures[1] == " juliacon2021"
+        @info "juliacon2021 was required" m.content m.author.username m.author.discriminator
+        _juliacon2021(c, m)
+    elseif matches.captures[1] == " now"
+        @info "now was required" m.content m.author.username m.author.discriminator
+        _now(c, m)
+    elseif matches.captures[1] == " today"
+        @info "today was required" m.content m.author.username m.author.discriminator
+        _today(c, m)
+    else
+        reply(c, m, "Sorry, are you looking for information on JuliaCon 2021? Please check out `con help`")
+    end
+    return nothing
+end
+
+function help_commander(c::Client, m::Message, ::Val{:julia_con})
+    reply(c, m, """
+        How to look for information about JuliaCon 2021 with the `con` command:
+        ```
+        con help
+        con juliacon2021
+        con now
+        con today
+        ```
+        `help` prints this message
+        `juliacon2021` greets the users and provide a link to the official website
+        `now` details the talks and events occuring right now on the different tracks
+        `today` lists the talks and event of the day
+        """)
+end
+
+function _juliacon2021(c::Client, m::Message)
+    reply(c, m, "Welcome to JuliaCon 2021! Find more information on https://juliacon.org/2021/.")
+    return nothing
+end
+
+function _now(c::Client, m::Message)
+    current_talks = JuliaCon.get_running_talks(; now=FAKENOW)
+    if isnothing(current_talks)
+        @info "There is no JuliaCon program today!"
+        return nothing
+    end
+
+    str = ""
+    for (track, talk) in current_talks
+        str *= """
+        $track
+        \t$(talk.title) ($(talk.type))
+        \t├─ $(JuliaCon.speakers2str(talk.speaker))
+        \t└─ $(talk.url)
+        """
+        # printstyled(track; bold=true, color=_track2color(track))
+    end
+    str *= "\n(Full schedule: https://pretalx.com/juliacon2021/schedule)"
+    reply(c, m, str)
+    return nothing
+end
+
+function split_pretty_table(str)
+    lines = split(str, "\n")
+    acc = "```\n"
+    strings = Vector{String}()
+    for line in lines
+        if sum(length, [acc, line]) ≤ 1996
+            acc *= "\n" * line
+        else
+            push!(strings, acc * "\n```")
+            acc = "```\n" * line
+        end
+    end
+    push!(strings, acc * "\n```")
+    return strings
+end
+
+function _today(c::Client, m::Message)
+    track_schedules = JuliaCon.get_today(; now=FAKENOW)
+    isnothing(track_schedules) && (@info "There is no JuliaCon program today!"; return nothing)
```

Comment:
This would look better in style in a full `if/end` block.

Also, comparing with `nothing` is more idiomatic as `x === nothing` than using `isnothing` function (VSCode linter told me about that before).

Reply:
Oh! And here I was thinking it was somewhat better to use `isnothing` haha. I learn every day.

You're right for the `if/else`. This will be handled by my PR in JuliaCon, so I won't touch it here!